### PR TITLE
Results grid options

### LINKED ISSUE #181: Feature Request: Modify Font for Results & Messages
Please add the ability to modify the font used in the results output and the messages output.

### LINKED ISSUE #761: Suggestion: Allow setting RowHeight in Results Grid View.
- SQL Operations Studio Version: 0.26.7

Steps to Reproduce: Run any query that returns results.

The Results Grid has very tall rows, showing about half the data density that SQL Server Management Studio would show in its Results view.

Suggestion: Add a Setting to set the RowHeight of the Results Grid.

### PR DESCRIPTION
**Pull request by anthonydresser**
Adds results grid options.
`resultsGrid.rowHeight`
`resultsGrid.cellPadding`
`resultsGrid.fontFamily`
`resultsGrid.fontSize`
`resultsGrid.fontWeight`
`resultsGrid.letterSpacing`

image

needs https://github.com/Microsoft/angular2-slickgrid/pull/28

fixes #761
fixes #181

maybe others, it's been asked for a lot

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     "@angular/router": "~4.1.3",
     "@angular/upgrade": "~4.1.3",
     "angular2-grid": "2.0.6",
-    "angular2-slickgrid": "git://github.com/Microsoft/angular2-slickgrid.git#1.3.9",
+    "angular2-slickgrid": "git://github.com/Microsoft/angular2-slickgrid.git#feat/setRowHeightRelease",
```
[the remark this answers is not in the document]
Comment (anthonydresser) [reply package.json]: yeah I need to merge it in and update the release

[the remark this answers is not in the document]
Comment (anthonydresser) [reply package.json]: https://github.com/Microsoft/angular2-slickgrid/pull/28

**Comment by mctunes** [comment]
Just installed Azure Data Studio 1.0.0, and tried modifying resultsGrid.rowHeight in user settings, with no effect, which I assume is because the code changes were reverted. Will you be revisiting this setting to try and get it implemented? It really would be a useful addition.

Many thanks for an awesome tool!

**Comment by Avertae** [comment]
Results row height is not configurable, default of 29 is too space eating, unfortunately.

**Pull request merged**